File: FCUBEAPI/FinanceMaster/Models/FinAccountsMasterGstModel.cs
using System.Data;

namespace FinanceMasters.Models
{
    public class FinAccountsMasterGstModel
    { 
        public string? AccountId { get; set; }
        public List<FinAccountsMasterGstDetail> finAccountsGstDetail { get; set; }

    }
    public class FinAccountsMasterGstDetail
    {
        public string? AccountId { get; set; }
        public string? Location { get; set; }
        public string? GstNo { get; set; }
        public string? Address1 { get; set; }
        public string? Address2 { get; set; }
        public string? Address3 { get; set; }
        public string? Address4 { get; set; }
        public string? City { get; set; }
        public string? StateCode { get; set; }
        public string? PinCode { get; set; }
        public string? MobileNo { get; set; }
        public string? Email { get; set; }
    }

}

File: OMKARAUTOAPP/src/app/models/finaccountsmastergstmodel.ts
export class Finaccountsmastergstmodel {
    accountId: string="";
    finAccountsGstDetail: Finaccountsmastergstdetails[] = [];
}

export class Finaccountsmastergstdetails {    
    accountId: string="";
    location : string="";
    gstNo: string="";
    address1 : string="";
    address2 : string="";
    address3 : string="";
    address4 : string="";
    city: string="";
    stateCode: string="";
    pinCode: string="";
    mobileNo : string="";
    email : string="";
}